JS/Lesson15/emrah/script.js

```diff
@@ -1,164 +1,201 @@
+console.log("------------------------");
 // 11. Use the `startsWith` method to check if strings below starts with a price, and print
 // the result to the console.
-const string1Exercise11 = '$9.99 - Sale! 50% off all items in store';
+const string1Exercise11 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise11 =
-	'Bundle deal: Save $50 when you buy two products together';
-const priceSymbol = '$';
-console.log('Exercise11: ', string1Exercise11.startsWith(priceSymbol));
-console.log('Exercise11: ', string2Exercise11.startsWith(priceSymbol));
+  "Bundle deal: Save $50 when you buy two products together";
+const priceSymbol = "$";
+console.log("Exercise11: ", string1Exercise11.startsWith(priceSymbol));
+console.log("Exercise11: ", string2Exercise11.startsWith(priceSymbol));
 
-console.log('------------------------');
+console.log("------------------------");
 // 12. Use the `endsWith` method to check if a string ends with a "!" (exclamation mark), and print the result to the console.
-const string1Exercise12 = '$9.99 - Sale! 50% off all items in store';
+const string1Exercise12 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise12 =
-	'Bundle deal: Save $50 when you buy two products together!';
+  "Bundle deal: Save $50 when you buy two products together!";
+const exclamationmarkSymbol = "!";
+console.log("Exercise12: ", string1Exercise12.endsWith(exclamationmarkSymbol));
+console.log("Exercise12: ", string2Exercise12.endsWith(exclamationmarkSymbol));
 
-console.log('Exercise12: ' /* Result here */);
-console.log('Exercise12: ' /* Result here */);
-
-console.log('------------------------');
+console.log("------------------------");
 // 13. Use the `includes` method to check if a string includes a "%" character and print
 // the result to the console.
-const string1Exercise13 = '$9.99 - Sale! 50% off all items in store';
+const string1Exercise13 = "$9.99 - Sale! 50% off all items in store";
 const string2Exercise13 =
-	'Bundle deal: Save $50 when you buy two products together!';
-const percentSymbol = '%';
-console.log('Exercise13: ', string1Exercise13.includes(percentSymbol));
-console.log('Exercise13: ', string2Exercise13.includes(percentSymbol));
+  "Bundle deal: Save $50 when you buy two products together!";
+const percentSymbol = "%";
+console.log("Exercise13: ", string1Exercise13.includes(percentSymbol));
+console.log("Exercise13: ", string2Exercise13.includes(percentSymbol));
 
-console.log('------------------------');
+console.log("------------------------");
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14
 // based on two indices and print it to the console.
 const stringExercise14 =
-	'Databases are an essential component of many applications.';
+  "Databases are an essential component of many applications.";
+
+const startIndex = 0;
+const endIndex = 9;
+const extractedWord = stringExercise14.substring(startIndex, endIndex);
```

Comment:
For this exercise and going forward, do not hardcode the start and end indexes like this. If you need to fine the word "Databases" and a string, what you should be doing is following:
- using `stringExercise14.indexOf("Databases")` find the start index
- using `"Databases".length` find the length of the string you're looking for
- find the end index by using start index and the length of the string

This way you'd get a dynamic indexes and your code would be reusable even if the initial sentece that you need to check changes.